gitbooks/features/platform.zh-CN.md

```
@@ -0,0 +1,75 @@
+
+**体积小。** 只有典型通信工具的几分之一。不到一秒启动，内存占用极少。
+
+**启动快。** 无需初始化浏览器引擎。立即就绪接受请求。
+
+**操作系统级安全。** 凭据保存在你平台的安全密钥链中：macOS Keychain、Windows Credential Manager、Linux Secret Service。敏感数据永不放在浏览器存储或明文文件中。本地记忆树的 SQLite 数据库位于你的工作区文件夹中，由你拥有。
+
+***
+
+## 架构概览
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)